app/common/templates/common/macros/collections.html

```diff
@@ -0,0 +1,180 @@
+{% from "govuk_frontend_jinja/components/back-link/macro.html" import govukBackLink %}
```

Comment:
We might want to split these into separate macro files at some point, but for now I think this is fine 👍